messages/device-added-notification.json

```diff
@@ -38,7 +38,419 @@
         "device": {
           "type": "object",
           "title": "Device",
-          "description": "Description of the device"
+          "description": "Description of the device",
+          "required": [
+            "id"
+          ],
+          "properties": {
+            "@context": {
+              "type": "string",
+              "title": "Device context",
+              "description": "Context of the device"
+            },
+            "@type": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "title": "Device types",
+                "description": "Types of the device"
+              }
+            },
+            "id": {
+              "type": "string",
+              "title": "Device ID",
+              "description": "ID of the device"
+            },
+            "title": {
+              "type": "string",
+              "title": "Device title",
+              "description": "Title of the Device"
+            },
+            "description": {
+              "type": "string",
+              "title": "Device Description",
+              "description": "Description of the device"
+            },
+            "properties": {
+              "type": "object",
+              "patternProperties": {
+                ".*": {
+                  "type": "object",
+                  "title": "Property",
+                  "description": "Description of the Property",
+                  "required": [
+                    "type",
+                    "visible"
```

Comment:
`visible` probably shouldn't be required.

Reply:
Done